stock_reserve/models/stock_reserve.py

```diff
@@ -106,9 +108,9 @@ def get_location_from_ref(self, ref):
         return location_id
 
     @api.model
-    def _default_picking_type_id(self):
-        ref = "stock.picking_type_out"
-        return self.env.ref(ref, raise_if_not_found=False).id
+    def _default_picking_type_id(self, company_id):
```

Comment:
This method may be executed directly by ORM, so arguments shouldn't change. Call `default_get` for `company_id` here or remove this extra method.